src/UI/Components/TreeTable/Inventory/AttributeHelper.ts

```diff
@@ -105,13 +126,26 @@ export class InventoryAttributeHelper
     let keys: AttributeNodeDict = {};
     const primaryKeys = Object.keys(subject).sort();
     primaryKeys.forEach((key) => {
+      let type;
+      if (this.service === undefined) {
+        type = "undefined";
```

Comment:
because without service there is no way to define type of that attribute, for some reason that creator allow to service to be undefined which don't make sense at first glance as without service there won't be attributes(I guess), I didn't investigate further, but yeah